luigi_pipeline/lib/model/seqr_mt_schema.py

```diff
@@ -8,11 +8,31 @@
 class SeqrSchema(BaseMTSchema):
 
     def __init__(self, *args, ref_data, clinvar_data, hgmd_data=None, **kwargs):
-        super().__init__(*args, **kwargs)
         self._ref_data = ref_data
         self._clinvar_data = clinvar_data
         self._hgmd_data = hgmd_data
 
+        # reuse:
+        #   self._ref_data[self.mt.row_key]
+        # for all annotations. We'll use the @property _selected_ref_data
+        # to access this value, and lazily cache it (so as not to compute
+        # it if it doesn't get get accessed after an update to self.mt
```

Comment:
this comment would be more helpful as a docstring for the `_selected_ref_data` property